kmcd-data.md

```diff
@@ -55,34 +64,26 @@ We model everything with arbitrary precision rationals, but use sort information
 
```

Comment:
Just above this line (commenting here since I can't comment outside of the diff), there are defs for `0Wad`, `1Wad`,  etc...seems like the rest of this PR systematically removes those in favor of `wad(0)`, etc; is there a reason to keep them?

Reply:
Good catch, I've removed them.